docusaurus/docs/learn/quickstarts/zac/index.md

```diff
@@ -1,122 +1,60 @@
-:::
-
-## Cloning From GitHub
-
-These steps are applicable to both the [Local - No Docker](/learn/quickstarts/network/local-no-docker.md) and the
-[hosted yourself](/learn/quickstarts/network/hosted.md) deployments. Do note, these steps expect you have the necessary
-environment variables established in your shell. If you used the default parameters, you can establish these variables
-using the file at `${HOME}/.ziti/quickstart/$(hostname)/$(hostname).env`. To deploy ZAC after following one of those guides,
-you can perform the following steps.
-
-1. Clone the ziti-console repo from github:
-
-   ```text
-   git clone https://github.com/openziti/ziti-console.git "${ZITI_HOME}/ziti-console"
-   ```
-
-1. Install Node modules:
-
-   ```text
-   cd "${ZITI_HOME}/ziti-console"
-   npm install
-   ```
-
-1. Build the core Angular library:
 
-   ```text
-   ng build ziti-console-lib
-   ```
-
-1. Build the Angular app:
+1. Ensure `ZITI_HOME` is set to your quickstart working directory and exported to forked processes. You can always restore the quickstart environment variables by sourcing the environment file.
 
    ```text
-   ng build ziti-console-node
+   source ${HOME}/.ziti/quickstart/$(hostname)/$(hostname).env
+   export ZITI_HOME
    ```
 
-1. Use the ziti-controller certificates for the console:
+1. Unzip the console in the controller's working directory.
 
-   Link a server certificate into the `ziti-console` directory. Your web browser won't recognize it, but it's sufficient for this exercise to have server TLS for your ZAC session.
-
-   ```text
-   ln -s "${ZITI_PKI}/${ZITI_CTRL_EDGE_NAME}-intermediate/certs/${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS}-server.chain.pem" "${ZITI_HOME}/ziti-console/server.chain.pem"
-   ln -s "${ZITI_PKI}/${ZITI_CTRL_EDGE_NAME}-intermediate/keys/${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS}-server.key" "${ZITI_HOME}/ziti-console/server.key"
-   ```
+    ```text
+    unzip -d ${ZITI_HOME}/zac ./ziti-console.zip
+    ```
 
-1. [Optional] Emit the console systemd file and update systemd to start the console (ZAC). If you have not sourced
-   [the Ziti helper script](https://get.openziti.io/ziti-cli-functions.sh) and you wish to have ZAC enabled with systemd,
-   you need to in order to get the necessary function. Either inspect the script and find the function, download and source it,
-   or source it directly from the internet (direct sourcing from internet shown below)
+1. In **${ZITI_HOME}/$(hostname -s).yaml**, add a web API binding `zac` in the list containing `edge-management`.
 
-   ```text
-   source /dev/stdin <<< "$(wget -qO- https://get.openziti.io/ziti-cli-functions.sh)"
-   createZacSystemdFile
-   sudo cp "${ZITI_HOME}/ziti-console.service" /etc/systemd/system
-   sudo systemctl daemon-reload
-   sudo systemctl enable --now ziti-console
-   ```
+    ```text
+          - binding: zac
+            options:
+              location: ${ZITI_HOME}/zac
```

Comment:
i think this is going to cause confusion. I'm pretty sure the quickstarts don't set ZITI_HOME variable. it would be better to show them something that makes it clear or it would be better to have them run a sed command that makes this change on their behalf and repalce the env var with the full path.

I have been using these three commands:
```
sed -i 's@#- binding@- binding@g' $ZITI_HOME/$(hostname).yaml
sed -i 's@#  options@  options@g' $ZITI_HOME/$(hostname).yaml
sed -i 's@#    location.*$@    location: "'$ZITI_HOME/zacs/v${ZAC_VERSION}'"@g' $ZITI_HOME/$(hostname).yaml
```

this will make their controller file look something like:
```
      - binding: zac
        options:
          location: "/home/ubuntu/.ziti/quickstart/ip-172-31-47-200/zacs/v3.2.3"
```